cob_datapipeline/catalog_move_alma_sftp_bw_to_s3_dag.py

```diff
@@ -0,0 +1,135 @@
+"""Airflow DAG to move bw files from alma-sftp to s3 and archives the bw ftp files."""
+from datetime import datetime, timedelta
+import logging
+from tulflow import tasks
+import airflow
+from airflow.models import Variable
+from airflow.providers.sftp.hooks.sftp import SFTPHook
+from airflow.operators.python_operator import PythonOperator
+from cob_datapipeline.operators.batch_sftp_to_s3_operator import BatchSFTPToS3Operator
+from cob_datapipeline.helpers import determine_most_recent_date
+from cob_datapipeline.tasks.xml_parse import update_variables
+
+ALMA_SFTP_CONNECTION_ID = "ALMASFTP"
+S3_CONN_ID = "AIRFLOW_S3"
+S3_BUCKET = Variable.get("AIRFLOW_DATA_BUCKET")
+
+# CREATE DAG
+DEFAULT_ARGS = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "start_date": datetime(2019, 5, 28),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "on_failure_callback": tasks.execute_slackpostonfail,
+    "retries": 5,
+    "retry_delay": timedelta(minutes=5),
+}
+
+DAG = airflow.DAG(
+    "catalog_move_alma_sftp_bw_to_s3",
```

Comment:
@dkinzer @ebtoner Any thoughts on making the name of this dag a little shorter? I feel like the name is too similar to the one for the alam files.  Maybe something like catalog_move_boundwith_to_s3?  I'm fine with either decision, just an idea.

Reply:
@sensei100 I'm fine with simplifying the names. Alternatively we could do "boundwith_move_alma_sftp_to_s3"? I think having a different prefix helps in distinguishing it more easily from "catalog_move_alma_sftp_to_s3"?